openqa_review/openqa_review.py

```diff
@@ -282,8 +282,14 @@ def get_test_bugref(entry):
     bugref = entry.find(id=re.compile("^bug-"))
     if not bugref:
         return {}
+    reference = re.search(r"\S+#((?:[A-Z]+-)?[0-9]+)", bugref.i["title"])
+    if reference:
+        reference = reference.group()
+    else:
+        sys.stderr.write("Error: Could not find bugref in '%s'\n" % bugref.i["title"])
+        reference = "???"
```

Comment:
We have a logging object which you should be able to use.
Also tests failed as this section is uncovered: https://app.travis-ci.com/github/os-autoinst/openqa_review/jobs/526835360#L252
As you handle the error already and provide the necessary results in the output I suggest to not treat it as an "error" from perspective of openqa-review as the mitigation is already automatically applied so we should rather make the reference text explicit. How about:

```suggestion
    reference = re.search(r"\S+#((?:[A-Z]+-)?[0-9]+)", bugref.i["title"])
    bugref = reference.group() if reference else "(unknown bugref in %s)" % bugref.i["title"]"
```

As an alternative to the above we could mark the section explicitly as acceptable to not be covered but I would very much prefer the above